styles/Home.module.scss

```diff
@@ -23,7 +9,13 @@
 }
 
 .homeCard {
-  color: green;
+  width: 45%;
+  height: 250px;
+
+  :hover {
+    transform: scale(1.01);
+    transition: 500ms ease;
+  }
 }
 
 .homeButton {
```

Comment:
The css class was deleted as it didn't make any change on the card. Just duplicated code without purpose